### PR TITLE
Refresh & cleanup freedesktop.org web URLs

### PR DESCRIPTION
**Pull request by sidt4**
Please refer commit logs.

### CODE REVIEW

**Comment by sidt4** [comment]
I'm not sure if the https://specifications.freedesktop.org/ URLs should have a `latest` or `1.0` / `1.1` value.

Marking as draft until this is resolved.

**Comment by ximion** [comment]
Whoops, I already adjusted a few of the links to have the right anchors. Can you rebase and add the remaining changes on top?

**Comment by sidt4** [comment]
> Whoops, I already adjusted a few of the links to have the right anchors.

I guess the changes are mutually exclusive.

**Comment by sidt4** [comment]
> I'm not sure if the [specifications.freedesktop.org](https://specifications.freedesktop.org/) URLs should have a latest or 1.0 / 1.1 value.

Using `latest`. Resolving thread.

File: compose/asc-hint-tags.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ AscHintTagStatic asc_hint_tag_list[] =  {
 	{ "ancient-metadata",
 	  AS_ISSUE_SEVERITY_WARNING,
 	  "The AppStream metadata should be updated to follow a more recent version of the specification.<br/>"
-	  "Please consult <a href=\"http://freedesktop.org/software/appstream/docs/chap-Quickstart.html\">the XML quickstart guide</a> for "
+	  "Please consult <a href=\"https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html\">MetaInfo quickstart guides</a> for "
```
Comment (ximion) [inline compose/asc-hint-tags.c:62]:
```suggestion
	  "Please consult <a href=\"https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html\">MetaInfo quickstart guide</a> for "
```

**Review by ximion — COMMENTED** [review]
Just one nitpick, then this looks good! Thank you a lot for looking into this! :-)

File: compose/asc-hint-tags.c (hunk 1 above)
Comment (sidt4) [reply compose/asc-hint-tags.c]: Sure.

**Pull request merged**